src/js/helpers/submissionGuideHelper.js

```diff
@@ -29,4 +29,37 @@ export const setSkipGuide = (skip_guide) => {
         });
 
     return deferred.promise;
+}
+
+export const getSubmissionPage = (submission_id) => {
+    const deferred = Q.defer();
+
+    const store = new StoreSingleton().store;
+
+    Request.get(kGlobalConstants.API + 'check_current_page/?submission_id=' + submission_id)
+        .end((err, res) => {
+            if (err) {
+                deferred.reject(err);
+            } else {
+                // Only skip the guide if the user wants to skip the guide
+                let pages = [
+                    '/404',
```

Comment:
0 returns if the submission files dont exist. Kaitlin said that that it should be fine to just redirect to a 404 page when no content exists
<img width="686" alt="screen shot 2017-05-16 at 4 03 34 pm" src="https://cloud.githubusercontent.com/assets/4334164/26125754/415891ec-3a51-11e7-8164-61fd44d25e6d.png">